.github/workflows/actions.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Install dependencies
         run: |
           pip install -e .[dev] \
-            -e benchadapt/python \
```

Comment:
Removed this for now because its only dep is benchclients, and it tries to install it from the `main` branch, which will fail until this is merged. We can put it back in a future PR, but it only really matters if it needs more dependencies